eZ/Publish/Core/Repository/Helper/FieldTypeRegistry.php

```diff
@@ -44,7 +44,7 @@ public function getFieldTypes()
         {
             if ( !$value instanceof SPIFieldType )
             {
-                return $this->getFieldType( $identifier );
+                $this->getFieldType( $identifier );
```

Comment:
Could maybe have inline comment saying function is reused to make sure loading is done.
@pspanja should add unit test for this so we avoid regression if someone changes getFieldType() and oversees this.